config/jest/testTheme.ts

```diff
@@ -153,19 +153,23 @@ export default {
     normalZIndex: '2',
     belowNormalZIndex: '1',
 
-    // ui elements
+    // page elements
     pageMainZIndex: '50',
 
-    tooltipZIndex: '8000',
-
-    dropdownMenuZIndex: '7500',
+    // generic page fixed elements
+    contextbarZIndex: '420',
```

Comment:
I added this missing one and re-ordered a bit the list to match the comment on top.